requirements.txt

```diff
@@ -1,4 +1,5 @@
 lxml==4.9.1
+matplotlib==3.7.2
```

Comment:
This needs to be added to the `docs/requuirements.txt` and matplotlib also needs to be added to `setup.py` on the `install_requires` line.

Reply:
Also, can you update the `CHANGELOG.md` please?